integrations/tensorflow-keras/neptune_tensorflow_keras/__init__.py

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import Optional
+
+from neptune.alpha import Experiment
+from neptune.alpha.exceptions import NeptuneException
+
+from ._version import get_versions
+__version__ = get_versions()['version']
+del get_versions
+
+# Note: we purposefully try to import `tensorflow.keras.callbacks.Callback`
+# before `keras.callbacks.Callback` because the former is compatible with both
+# `tensorflow.keras` and `keras`, while the latter is only compatible
+# with `keras`. See https://github.com/keras-team/keras/issues/14125
+try:
+    from tensorflow.keras.callbacks import Callback
+except ImportError:
+    try:
+        from keras.callbacks import Callback
+    except ImportError:
+        msg = """
+        keras package not found. 
+
+        As Keras is now part of Tensorflow you should install it by running
+            pip install tensorflow"""
+        raise ModuleNotFoundError(msg) # pylint:disable=undefined-variable
+
+
+class NeptuneCallback(Callback):
+    """Logs Keras metrics to Neptune.
+
+    Goes over the `last_metrics` and `smooth_loss` after each batch and epoch
+    and logs them to Neptune.
+
+    If experiment parameter is not provided, Neptune will attempt to self-initialize using
+    `project` and  `api_token` parameters
+
+    See the example experiment here https://ui.neptune.ai/shared/keras-integration/e/KERAS-23/logs
+
+    Args:
+        experiment: `neptune.Experiment`:
```

Comment:
`neptune.new.Experiment`?

Reply:
we will do a single, bulk edit alpha -> new in future. Utlill it's done, it should be alpha